release_notes.md

```diff
@@ -0,0 +1,67 @@
+ # Release notes
+
+- Woof Mobile APP
+- Release notes version: 1
+- Release note date: May. 4, 2023
+
+All notable changes to this application will be documented in this file.
```

Comment:
From this description I don't understand the difference between changelog and release notes.

Reply:
At this stage of the application development almost there is no difference between this files, because there is no much functionality in the application. But then, in release_notes.md file will be detailed the changes, upgrades, additional features and bug fixes.